tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,85 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  
+Having kernels configured and packaged for specific purposes gives the Jupyter environment some flexibility.
+A kernel doesn't need to be configured to serve every purpose imaginable. Several kernels can be made, with none
```

Comment:
Can we reword this from the user's perspective?

> You will likely not use a single kernel for every purpose, instead you can choose which kernel best suits your workflow.

Reply:
Reworded it to remove some technical speak and add focus to the user.